src/main/java/seedu/address/logic/parser/AddPersonCommandParser.java

```diff
@@ -51,7 +52,7 @@ public AddPersonCommand parse(String args) throws ParseException {
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
 
-        Person person = new Person(name, phone, email, nusNet, address, tagList);
+        Person person = new Person(name, phone, email, nusNet, address, new HashSet<>(), tagList);
```

Comment:
Perhaps we can move `new HashSet<>()` into the person constructor, since every time we add a person via the command, the new person have no attendance marked? We can add another person constructor with a different signature, and use it here:
```suggestion
        Person person = new Person(name, phone, email, nusNet, address, tagList);
```

Reply:
Implemented in e6bf6760c4ee6ccd0246bec9b03915d147c6c2b2